docs/reviewing.md

```diff
@@ -113,3 +113,30 @@ When components are defined in the `package_info` in `conanfile.py` the followin
 
 - use the `cmake_find_package` or `cmake_find_package_multi` generators in `test_package/conanfile.py`
 - corresponding call to `find_package()` with the components _explicitly_ used in `target_link_libraries`
+
+### Recommended feature options names
+
+It's often needed to add options to toggle specific library features on/off. Sometimes, it's preferred to have options enabled by default, sometimes disabled by default. In order to avoid the fragmentation, we recommend to use the following naming conventions for such options:
+
+- enable_xxx / disable_xxx
+- with_xxx / without_xxx
+- use_xxx
```

Comment:
I've never seen a `use_xxx` and searching quickly did not produce any

```suggestion
```

Reply:
there are at least some usages:
```
./recipes/sqlite3/all/conanfile.py:34:        "use_alloca": [True, False],

./recipes/openblas/all/conanfile.py:21:        "use_thread": [True, False],

./recipes/ceres-solver/all/conanfile.py:17:               "use_glog":  [True, False], #TODO Set to true once gflags with nothreads=False binaries are available. Using MINILOG has a big performance drawback.
./recipes/ceres-solver/all/conanfile.py:18:               "use_gflags": [True, False],
./recipes/ceres-solver/all/conanfile.py:19:               "use_custom_blas": [True, False],
./recipes/ceres-solver/all/conanfile.py:20:               "use_eigen_sparse": [True, False],
./recipes/ceres-solver/all/conanfile.py:21:               "use_TBB": [True, False],
./recipes/ceres-solver/all/conanfile.py:22:               "use_CXX11_threads": [True, False],
./recipes/ceres-solver/all/conanfile.py:23:               "use_CXX11": [True, False],
./recipes/ceres-solver/all/conanfile.py:24:               "use_schur_specializations": [True, False]}

./recipes/openscenegraph/all/conanfile.py:28:        "use_fontconfig": [True, False],

./recipes/libsodium/1.0.18/conanfile.py:22:        "use_soname" : [True, False],

./recipes/jansson/all/conanfile.py:18:        "use_urandom": [True, False],
./recipes/jansson/all/conanfile.py:19:        "use_windows_cryptoapi": [True, False]

./recipes/fruit/all/conanfile.py:18:               "use_boost": [True, False],

./recipes/gtsam/all/conanfile.py:20:               "use_quaternions": [True, False],

./recipes/date/all/conanfile.py:19:               "use_system_tz_db": [True, False],
./recipes/date/all/conanfile.py:20:               "use_tz_db_in_dot": [True, False]}

```